gravitee-apim-console-webui/src/management/application/details/subscriptions/application-subscriptions-list.component.html

```diff
@@ -34,7 +34,7 @@ <h2>{{ $ctrl.listLabel }}</h2>
         <tbody md-body>
           <tr md-row ng-repeat="subscription in $ctrl.subscriptions.data track by subscription.id">
             <td md-cell>
-              <a ng-click="$ctrl.navigateToSubscription(subscription.id)">
+              <a ng-click="$ctrl.navigateToSubscription(subscription.id)" style="cursor: pointer">
```

Comment:
Actually we have the same "error" everywhere in the code.
I think we should think of a larger solution